packages/modules/update_soc.py

```diff
@@ -72,6 +72,9 @@ def _get_threads(self) -> Tuple[List[Thread], List[Thread]]:
                     if ev.data.get.fault_state != 0 or ev.data.get.fault_str != NO_ERROR:
                         Pub().pub(f"openWB/set/vehicle/{ev.num}/get/fault_state", 0)
                         Pub().pub(f"openWB/set/vehicle/{ev.num}/get/fault_str", NO_ERROR)
+                        Pub().pub(f"openWB/set/vehicle/{ev.num}/get/soc", None)
+                        Pub().pub(f"openWB/set/vehicle/{ev.num}/get/soc_timestamp", None)
+                        Pub().pub(f"openWB/set/vehicle/{ev.num}/get/range", None)
```

Comment:
Ich persönlich fände es besser, wenn die letzten bekannten Stände in der GUI angezeigt blieben. 

Insbesondere `/soc_timestamp", None)` halte ich für problematisch, weil danach ja die SOC in den Graphen einsortiert werden müssen, oder nicht?